src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.xml

```diff
@@ -60,7 +60,7 @@
 
         <!-- transports -->
         <service id="messenger.transport_factory" class="Symfony\Component\Messenger\Transport\TransportFactory">
-            <argument type="tagged" tag="messenger.transport_factory" />
+            <argument type="tagged_iterator" tag="messenger.transport_factory" />
```

Comment:
this means FWB now needs DI v4.4
this is why tests fail: the composer.json needs an update